src/app/organizations/manage/policy-edit.component.html

```diff
@@ -21,27 +21,68 @@ <h2 class="modal-title" id="policiesEditTitle">{{'editPolicy' | i18n}} - {{name}
                     </div>
                 </div>
                 <ng-container *ngIf="type === policyType.MasterPassword">
-                    <div class="form-group">
-                        <label for="masterPassMinLength">{{'minLength' | i18n}}</label>
-                        <input id="masterPassMinLength" class="form-control" type="number" name="MasterPassMinLength"
-                            [(ngModel)]="masterPassMinLength">
+                    <div class="row">
+                        <div class="col-6 form-group">
+                            <label for="masterPassMinComplexity">{{'minComplexityScore' | i18n}}</label>
+                            <input id="masterPassMinComplexity" class="form-control" type="number"
+                                name="MasterPassMinComplexity" [(ngModel)]="masterPassMinComplexity" min="0" max="4">
+                        </div>
+                        <div class="col-6 form-group">
+                            <label for="masterPassMinLength">{{'minLength' | i18n}}</label>
+                            <input id="masterPassMinLength" class="form-control" type="number"
+                                name="MasterPassMinLength" [(ngModel)]="masterPassMinLength">
+                        </div>
+                    </div>
+                    <div class="form-check">
+                        <input class="form-check-input" type="checkbox" id="masterPassRequireUpper"
+                            [(ngModel)]="masterPassRequireUpper" name="MasterPassRequireUpper">
+                        <label class="form-check-label" for="masterPassRequireUpper">A-Z</label>
+                    </div>
+                    <div class="form-check">
+                        <input class="form-check-input" type="checkbox" id="masterPassRequireLower"
+                            [(ngModel)]="masterPassRequireLower" name="MasterPassRequireLower">
+                        <label class="form-check-label" for="masterPassRequireLower">a-z</label>
```

Comment:
Placeholder for eventual string resource

Reply:
I think I misunderstood you in chat. These actually won't need a string resource. They are this way in all of our apps since it doesn't really change from language to language.